proto/storage/alert.proto

```diff
@@ -96,62 +95,62 @@ message Alert {
 
   message ProcessViolation {
     string message = 1;
-    repeated ProcessIndicator processes = 2 [(gogoproto.moretags) = 'sql:"ignore-fks"'];
+    repeated ProcessIndicator processes = 2; // @gotags: sql:"ignore-fks"
   }
 
   message Enforcement {
-    storage.EnforcementAction action = 1 [(gogoproto.moretags) = 'search:"Enforcement"'];
+    storage.EnforcementAction action = 1; // @gotags: search:"Enforcement"
     string message = 2;
   }
 
-  string id = 1 [(gogoproto.moretags) = 'search:"Alert ID,hidden" sensorhash:"ignore" sql:"pk,type(uuid)"']; // Internal use only
```

Comment:
It would be good to keep previous comments. I think they can be just added as a comment before (or after) `@gotags`
Like this:
```
 string id = 1; // Internal use only // @gotags: search:"Alert ID,hidden" sensorhash:"ignore" sql:"pk,type(uuid)"
```
But I didn't test that.

If not, we can move them before the field:
```
// Internal use only
string id = 1; // @gotags: search:"Alert ID,hidden" sensorhash:"ignore" sql:"pk,type(uuid)"
```

There are other places that have comments, but not many. Maybe a dozen or so -> that means it should be hard to adjust them manually.

Here is the list of files that I have encountered during review:
```
proto/storage/cve.proto
proto/storage/deployment.proto
proto/storage/image_component.proto
proto/storage/kube_event.proto
proto/storage/node_component.proto
proto/storage/service_identity.proto
```